inst/qml/common/ui/DataSplit.qml

```diff
@@ -94,6 +94,7 @@ Section
 				addEmptyValue: 					true
 				placeholderText: 				qsTr("None")
 				info:							qsTr("The variable in the data set that is used as the test set indicator.")
+				source:							[ { model: columnsModel, use: "type=scale"} ]
```

Comment:
Better is to use the allowedColumns property: `allowedColumns: "scale"`
Then also nominal variables will be also authorized. Would be strange to refuse variable as `contBinom`.

Reply:
No they are not allowed and will crash the analysis

Reply:
OK, nice

Reply:
The `allowedColumns` shiould tranform the variable into a scale.

Reply:
I changed it